.teamcity/src/main/kotlin/configurations/Gradleception.kt

```diff
@@ -48,7 +48,11 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(sta
     val dogfoodTimestamp1 = "19800101010101+0000"
     val dogfoodTimestamp2 = "19800202020202+0000"
     val buildScanTagForType = buildScanTag("Gradleception")
-    val defaultParameters = (buildToolGradleParameters() + listOf(buildScanTagForType) + "-Porg.gradle.java.installations.auto-download=false").joinToString(separator = " ")
+    val buildScanTagForGroovy4 = buildScanTag("Groovy 4.x")
```

Comment:
I suggest to use something like `Groovy4`, without space, otherwise it would be a problem either in passing `-Dscan.tag=Groovy 4.x` or querying in build scan web page.

Reply:
Though both of them works now.

Reply:
I agree, removing spaces/punctuation is better.